app/views/comments/_mentionable_users.html.erb

```diff
@@ -6,5 +6,5 @@
       value: user.email,
       avatar_url: avatar_url(user)
     }
-  end.to_json %>">
+  end.to_json %>" data-fallback-image="<%= image_url(AvatarHelper::DEFAULT_PROFILE_IMAGE) %>">
```

Comment:
we're stretching ERB inlining in this template, maybe time to extract to a `content_tag :meta` call

Reply:
I figured there would be an easy way to manage this. And there isn't.

TIL: `<%= content_for :meta, {any: :options} %>`

will produce `<meta>{any: :options}</meta>`

And it doesn't look like rails has *any* included methods for adding meta tags nicely. So as much as I'd like to fix it up more, lets merge as-is and worry about prettying it up later on.